LiskHQ/lisk-desktop
# Display the list of top accounts - closes #2454

### What issue have I solved?

-- #2454

### How have I implemented/fixed it?

For this feature, I had to do the following.

- [x] Add a new tab for Accounts in Monitor page.
- [x] Add a new component called Accounts table to show the information.
- [x] Create a screen component called Accounts under monitor directory, where AccountsTable is render.
- [x] Add a new function to fetch the top accounts in the liskService file under API directory.
- [x] Add unit test for Accounts component.

### How has this been tested?


1. Run the app.
2. Login to Mainnet or Testnet.
3. Go to Monitor page.
4. In the tabs options select Accounts (Top Accounts).
5. See that everything works as should be even Load More button

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
